packages/common/http/src/fetch.ts

```diff
@@ -70,13 +70,37 @@ export class FetchBackend implements HttpBackend {
   private readonly fetchImpl =
     inject(FetchFactory, {optional: true})?.fetch ?? ((...args) => globalThis.fetch(...args));
   private readonly ngZone = inject(NgZone);
+  private readonly aborters = new Set<AbortController>();
+
+  constructor() {
+    const destroyRef = inject(DestroyRef);
+    destroyRef.onDestroy(() => {
+      if (this.aborters.size === 0) {
+        return;
+      }
+
+      const aborters = Array.from(this.aborters.values());
+      this.aborters.clear();
```

Comment:
nit: we can save a few bytes here by avoiding `this.aborters` (since property reads are not minified):
```suggestion
    destroyRef.onDestroy(() => {
      const aborters = this.aborters;
      if (aborters.size === 0) {
        return;
      }

      const abortersAsArray = Array.from(aborters.values());
      aborters.clear();
```